Dictionary.json

```diff
@@ -6026,6 +6026,39 @@
 		"Tips5" : "https://www.viralpatel.net/category/spring/ ",
 		"Tips6" : "https://springframework.guru/free-spring-framework-tutorials/ "
 	},
+
+	{
+		"Title" : "Sql",
```

Comment:
Change 'Sql' to 'SQL'

Reply:
Sorry for late response , was covid strucked.